packages/tc-schema-sdk/data-accessors/type.js

```diff
@@ -133,25 +133,53 @@ const getType = function(
 	}
 
 	if (withRelationships) {
-		Object.entries(typeSchema.properties).forEach(([propName, def]) => {
-			if (def.relationship || def.cypher) {
-				if (def.hidden) {
-					delete typeSchema.properties[propName];
+		Object.keys(typeSchema.properties).forEach(key => {
+			let definition;
+			try {
+				definition = this.getRelationshipType(typeSchema.name, key, {
+					primitiveTypes,
+					includeMetaFields,
+				});
+			} catch (error) {
```

Comment:
What are the scenarios when this errors?

Reply:
I implemented `getRelationshipType` accessor as same as `getType()`, which throws error if type is not found.
So we should catch that error and it reckons `relationship is not found` - but we can return `null` and compare -- it is more clearer